modules/users/client/components/AboutMe.component.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import sanitizeHtml from 'sanitize-html';
+import { withNamespaces } from '@/modules/core/client/utils/i18n-angular-load';
+{/* import { Trans } from 'react-i18next'; */}
+import '@/config/lib/i18n';
+import PropTypes from 'prop-types';
+import { limitTo, plainTextLength } from './utils/filters';
+
+export class AboutMe extends Component {
+  constructor(props) {
+    super(props);
+    this.changeProfileDescriptionToggle = this.changeProfileDescriptionToggle.bind(this);
+
+    this.state = {
+      profileDescriptionToggle: false
+    };
+  }
+  changeProfileDescriptionToggle(){
+    this.setState((prevState) => ({
+      profileDescriptionToggle: !prevState.profileDescriptionToggle
+    }));
+  }
+
+  render(){
+    const { t, profile, isSelf, appSettings } = this.props;
+    return (<>
+      <section className="panel panel-default">
+        <header className="panel-heading">
+          {t('About me')}
+        </header>
+        <div className="panel-body">
+          {/* Short descriptions */}
+          {(profile.description.length < 2400 || this.state.profileDescriptionToggle) &&
+          <div dangerouslySetInnerHTML={{__html: sanitizeHtml(profile.description)}}></div>}{/* TODO Can we trust this HTML?*/}
+
+          {/* Long descriptions */}
+          {profile.description.length >= 2400 && !this.state.profileDescriptionToggle &&
+          <div
+            className="panel-more-wrap">
+            <div className="panel-more-excerpt" dangerouslySetInnerHTML={{__html: sanitizeHtml(limitTo(profile.description, 2400))}}></div>
+            <div className="panel-more-fade" onClick={this.changeProfileDescriptionToggle}>{t('Show more...')}</div>
+          </div>}
+
+          {/* If no description, show deep thoughts... */}
+          {(!profile.description || profile.description === '') &&
+          <blockquote className="profile-quote"
+            aria-label={t('Member has not written description about themself.')}>
+          “Everyone is necessarily the hero of their own life story.”
+          </blockquote>}
+        </div>
+
+        {/* User watching their own profile and it's too short */}
+        {(isSelf && (!profile.description || plainTextLength(profile.description) < appSettings.profileMinimumLength)) &&
+        <footer className="panel-footer">
+          <p className="lead">
+            {t('Your profile description should be longer so that you can send messages.')}
```

Comment:
For some reason I see this piece at (everybody with no description)'s profile. As if isSelf is always true.